pkgs/top-level/python-packages.nix

```diff
@@ -2832,8 +2832,6 @@ self: super: with self; {
 
   EasyProcess = callPackage ../development/python-modules/easyprocess { };
 
-  easysnmp = callPackage ../development/python-modules/easysnmp { };
-
```

Comment:
Nitpick: can add a `throw` to aliases

Reply:
Was only supported up to python 3.6 which was removed long ago.